### PR TITLE
feat: Pass issuer to verifyJWS

### PR DESCRIPTION
**Pull request by ukstv**
DID.verifyJWS accepts issuer parameter now. TileDocumentHandler does not have to verify kid-controller correspondence by itself.

### CODE REVIEW

File: packages/stream-tile-handler/src/tile-document-handler.ts [hunk 1]
```diff
@@ -193,21 +193,15 @@ export class TileDocumentHandler implements StreamHandler<TileDocument> {
...
-      result = await context.did.verifyJWS(commit, {
+      const verification = await context.did.verifyJWS(commit, {
         atTime: meta.timestamp,
+        issuer: did,
         disableTimecheck: meta.disableTimecheck,
       })
+      return verification.didResolutionResult
     } catch (e) {
       throw new Error('Invalid signature for signed commit. ' + e)
     }
```
Comment (oed) [inline packages/stream-tile-handler/src/tile-document-handler.ts:205]: Why is this wrapped in a try catch and an error is rethrown? Seems better to just let the error from *verifyJWS* propagate?

File: packages/stream-tile-handler/src/tile-document-handler.ts [hunk 2]
```diff
@@ -193,21 +193,15 @@ export class TileDocumentHandler implements StreamHandler<TileDocument> {
...
   ): Promise<DIDResolutionResult> {
-    let result
     try {
-      result = await context.did.verifyJWS(commit, {
+      const verification = await context.did.verifyJWS(commit, {
         atTime: meta.timestamp,
+        issuer: did,
         disableTimecheck: meta.disableTimecheck,
       })
+      return verification.didResolutionResult
```
Comment (oed) [inline packages/stream-tile-handler/src/tile-document-handler.ts:202]: I guess there's not really any reason to return anything from this method?

**Review by oed — COMMENTED** [review]
makes sense

File: packages/stream-tile-handler/src/tile-document-handler.ts (hunk 1 above)
Comment (ukstv) [reply packages/stream-tile-handler/src/tile-document-handler.ts:205]: Was here before. Decided not to change this.

File: packages/stream-tile-handler/src/tile-document-handler.ts (hunk 2 above)
Comment (ukstv) [reply packages/stream-tile-handler/src/tile-document-handler.ts:202]: Was here before. Decided not to change this.

Comment (ukstv) [reply packages/stream-tile-handler/src/tile-document-handler.ts]: Changed.

File: packages/stream-tile-handler/src/tile-document-handler.ts (hunk 1 above)
Comment (ukstv) [reply packages/stream-tile-handler/src/tile-document-handler.ts]: Changed.

**Review by oed — APPROVED** [review]
lgtm, why is this still a draft?

**Comment by ukstv** [comment]
> lgtm, why is this still a draft?

Discovered uint8arrays rug pull, and made it a draft to not confuse, until I am sure tests pass all right with jest resolver fix.

**Comment by ukstv** [comment]
Going to undraft it when tests are green.

**Comment by ukstv** [comment]
Undrafted. All green.

**Pull request merged**